report_client/views.py

```diff
@@ -18,10 +16,13 @@
 # Create your views here.
 class ReportsView(generics.CreateAPIView):
     def post(self, request, *args, **kwargs):
-        report = ParseDict(request.data, employee_report_pb2.Report())
-        response = stub.CreateReport(request=report)
-        json = MessageToJson(response, preserving_proto_field_name=True)
-        return Response(json, status=status.HTTP_201_CREATED)
+        try:
+            report = ParseDict(request.data, employee_report_pb2.Report())
+            response = stub.CreateReport(request=report)
+            json = MessageToJson(response, preserving_proto_field_name=True)
+            return Response(json, status=status.HTTP_201_CREATED)
+        except grpc.RpcError as e:
+            return Response(str(e), status=status.HTTP_500_INTERNAL_SERVER_ERROR)
```

Comment:
Why you don't want to return 400 error with error explanation? Raising 500 error to the external service not a good idea. Try to handle different error types.